backpack/extensions/secondorder/diag_hessian/conv2d.py

```diff
@@ -9,3 +12,12 @@ def __init__(self):
             N=2,
             params=["bias", "weight"],
         )
+
+
+class BatchDiagHConv2d(BatchDiagHConvND):
```

Comment:
docstring

Reply:
Do you mean we need to add docstring? Because `DiagGGN` also does not have and it will be inconsistent to add it here. Maybe we should do a seperate PR and add docstring to every individual `Conv1,2,3d` functions in 2nd order

Reply:
Yes. As discussed internally, documentation should be included for new and modified functions to improve the internal documentation.